### ISSUE TITLE
Documentation: Number of WCAG 2.0 Level A rules in axe-core exceeds the spec

### ISSUE THREAD

**Issue by mohanraj-r** [dequelabs/axe-core#2250]
Expectation: The number of WCAG 2.0 Level A rules in [axe-core/rule-descriptions.md](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md)
be less than or equal to the number as specified in [Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/WAI/GL/WCAG20/) 

Actual: The number of rules tagged `wcag2a` in [axe-core/rule-descriptions.md](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md) is 50, which exceeds the number of `(Level A)` rules in [Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/WAI/GL/WCAG20/) spec doc - which is 25.

Motivation: Would like to know what percentage of WCAG rules by different versions and levels is supported by axe-core. But seems like something weird is going on with axe-core seeming to support more WCAG 2 Level A rules than there are in the spec.

**Comment by mohanraj-r** [comment]
In addition to the numbers in terms of coverage of WCAG rules by levels, is there a doc which lists the limitations of axe-core - describing what axe (or any automated accessibility testing tool) can't test for?

**Comment by straker** [comment]
The problem is that axe rules do not map 1:1 to WCAG rules. We write rules in such a way as to give helpful remediation messages when things fail.

For example, `svg-img-alt`, `role-img-alt`, and `img-alt` rules each do the same thing (check for alternative text on image like elements) but each one has a different way to solve it. So we have 3 rules that all test WCAG 1.1.1.

As for the limitations, as far as I know we don't have such a list handy.
